### PR TITLE
feat: add vue2 support

### PR DESCRIPTION
**Pull request by Toilal**
#8 

I still have to work on this, maybe `vite-plugin-vue2` requires some changes too.

### CODE REVIEW

**Comment by Toilal** [comment]
I got it :).

It now works with both vue2 and vue3 plugin, without even referencing any plugin name. 

It should also be more performant : In current version, a markdown file is processed **two times** instead of once by the vue plugin on HMR. 

The first invocation is caused by this instruction from md plugin :

https://github.com/antfu/vite-plugin-md/blob/b7fc74a63c1a3fdd7657a48f51a1cf008fabaa44/src/index.ts#L27-L32

But a second invocation is still performed by the vue plugin itself, because `*.md` is included in the vue plugin configuration. 

```typescript
// From README.md
export default {
  plugins: [
    Vue({
      include: [/\.vue$/, /\.md$/], // <--
    }),
    Markdown()
  ],
}
```

This second invocation can't work properly, as the file is processed with raw unconverted markdown content. `@vue/template-sfc` seems to be more lenient and doesn't fail, but `vue-template-compiler` used inside the vue2 plugin fails because of this.

File: src/index.ts [hunk 1]
```diff
@@ -7,29 +7,19 @@ function VitePluginMarkdown(userOptions: Options = {}): Plugin {
...
   return {
     name: 'vite-plugin-md',
     enforce: 'pre',
-    configResolved(config) {
-      vuePlugin = config.plugins.find(p => p.name === 'vite:vue')!
-      if (!vuePlugin)
-        throw new Error('[vite-plugin-md] no vue plugin found, do you forget to install it?')
-    },
     transform(raw, id) {
       if (id.endsWith('.md'))
```
Comment (Toilal) [inline src/index.ts:14]: In think this extension should also be configurable, but i'll make another pull request when this one is merged.

Comment (Toilal) [reply src/index.ts:14]: something like `include`/`exclude` regexes like in vue plugin.

```typescript
include?: string | RegExp | (string | RegExp)[]
exclude?: string | RegExp | (string | RegExp)[]
```

**Comment by antfu** [comment]
Great, thansk!

**Pull request merged**